linked-list.rb

```diff
@@ -63,12 +63,36 @@ def display
    end
 
    def include?(key)
+     current = @head
+     full_list = []
+     while current.next_node != nil
+         full_list += [current.value.to_s]
+         (return true) if current.value == key
+         current = current.next_node
+     end
+     return false
    end
 
    def size
+     current = @head
+     count = 1
+     full_list = []
+     while current.next_node != nil
+         full_list += [current.value.to_s] #not necessary but nice
+         current = current.next_node
+         count += 1
+     end
+     return count
    end
 
    def max
+     current = @head
+     max = 0
+     while current.next_node != nil
+       (max += current.value) if current.value > max
+         current = current.next_node
+     end
+     return max
    end
```

Comment:
I'm not exactly sure what this method is doing, but it's not doing what I intended, which is to return the largest single value in the linked list.